disnake/abc.py

```diff
@@ -1287,6 +1287,11 @@ async def create_invite(
         :class:`.Invite`
             The newly created invite.
         """
+        if isinstance(target_application, PartyType):
+            utils.warn_deprecated(
+                "target_application parameter is deprecated and will be removed in future version",
+                stacklevel=2,
+            )
```

Comment:
`target_application` is unlikely to be removed, it's still a valid and documented API field.
It should however now support `Object`s as an alternative (with accompanying changelog and `versionchanged` documentation), and `PartyType` for backwards compatibility until the next version

Reply:
This doesn't seem to be resolved yet (and neither is the comment above)?

At least for this comment, something like [this](https://gist.github.com/shiftinv/e8f091e813e5a598af10e0c01e538a08) could work, I believe.